services/wireguard/resources/allocator_unix.go

```diff
@@ -126,7 +124,12 @@ func (a *Allocator) AllocatePort() (int, error) {
 	a.mu.Lock()
 	defer a.mu.Unlock()
 
-	for i := a.portMin; i < a.portMax; i++ {
+	if err := a.initializePortMin(); err != nil {
```

Comment:
Are we checking only min port availability, but continue increasing it without any check?

Reply:
Yeah, that was the plan, since this is only for testing purposes to reduce the number of clashes when running provider nodes from the office. But I'll change to all randomized ports.